controllers/jenkins/pipeline/metadata_converter.go

```diff
@@ -27,6 +27,28 @@ func convertPipeline(jobPipeline *job.Pipeline) *pipeline.Metadata {
 	}
 }
 
+// ParameterDefinitionTypeMap is for simplifying parameter definition type.
+var ParameterDefinitionTypeMap = map[string]string{
+	"StringParameterDefinition":   "string",
+	"ChoiceParameterDefinition":   "choice",
+	"TextParameterDefinition":     "text",
+	"BooleanParameterDefinition":  "boolean",
+	"FileParameterDefinition":     "file",
+	"PasswordParameterDefinition": "password",
```

Comment:
we already have this map. Please consider reusing it.

https://github.com/kubesphere/ks-devops/blob/4904f4ebbc3f88ae77402658f3e4e23e479f8b49/pkg/client/devops/jenkins/constants.go#L44

Reply:
Those map data cannot be merged into 

https://github.com/kubesphere/ks-devops/blob/4904f4ebbc3f88ae77402658f3e4e23e479f8b49/pkg/client/devops/jenkins/constants.go#L38-L45

This behaviour would break logic as below

https://github.com/kubesphere/ks-devops/blob/4904f4ebbc3f88ae77402658f3e4e23e479f8b49/pkg/client/devops/jenkins/pipeline_internal.go#L184-L185

Reply:
And [the test error](https://github.com/kubesphere/ks-devops/runs/4307589828?check_suite_focus=true) was caused by the merging.

Reply:
I didn't request you to merge them together. Please consider to reuse them.

Reply:
Got it.